src/rules/__tests__/lowercase-name.test.ts

```diff
@@ -225,3 +225,37 @@ ruleTester.run('lowercase-name with ignore=it', rule, {
   ],
   invalid: [],
 });
+
+ruleTester.run('uppercase-name with ignore=top', rule, {
+  valid: [
```

Comment:
Could you add a few muti-top level test cases, i.e:

```
describe('MyFirstClass', () => {
  ...
});

describe('MySecondClass', () => {
  ...
});
```

Ideally a couple would be nice, w/ a mix of single & multiple inner methods as well :)